package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "version": "7.0.0",
+  "version": "7.0.1",
```

Comment:
i think this should be 7.1.0 since this is a feature release. (https://semver.org/)